### PR TITLE
Hello Microverse Project: Addition of Linters, ReadMe file and Index.html file

### PR DESCRIPTION
**Pull request by AsciencioAlex**
creation of a readme file
creation of an index.html file
addition of linters to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alex Asciencio**
+- GitHub: [@githubhandle](https://github.com/Asciencioalex)
+- Twitter: [@twitterhandle](https://twitter.com/@Codenetech)
+- LinkedIn: [LinkedIn](https://linkedin.com/alex-asciencio-413612b9/)
+
```
Comment (Efiamotu-1) [inline README.md]: - [ ] The links you have provided for twitter and linkedIn are broken. Please kindly ensure that you are providing the appropraite links and if you do not have an account please open one and put in the correct link. It is not professional to have broken links in your readme file :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,187 @@
...
+- [ ] **CSS**
+- [ ] **Javascript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (Efiamotu-1) [inline README.md]: - [ ] This link is also broken please provide the appropriate link to the issues tab for this repository :+1:

File: README.md [hunk 3]
```diff
@@ -1 +1,187 @@
...
+- **headers**
+- **paragraphs**
+- **html** 
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon
```
Comment (Efiamotu-1) [inline README.md]: - [ ] If your project doesn't have a live demo there is no need for this section, so you can remove it. :+1: 
- [ ] In removing it make sure to make the same changes to the `Table of content` where you have `live demo` and remove it too :+1: :sparkles:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @AsciencioAlex , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,182 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alex Asciencio**
+- GitHub: [@githubhandle](https://github.com/Asciencioalex)
+- Twitter: [@twitterhandle](https://twitter.com/Codnetech)
+- LinkedIn: [LinkedIn](https://linkedin.com/alex-asciencio-413612b9/)
+
```
Comment (tresorsawasawa) [inline README.md]: - [ ] As mentioned by the previous reviewer  to provide working links for twitter and linkedin, kindly ensure that you update them and use the working ones. Right now the twitter link opens a profile different to yours and the linkedIn one opens a broken page. If you don't have accounts I **suggest you create ones and use working links** or **you can remove them from now and add them them later when you will create accounts for each one**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @AsciencioAlex 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (AsciencioAlex) [reply README.md]: @tresorsawasawa the twitter account is mine not anyone's else.

**Review by AbelG101 — APPROVED** [review]
# Hi @AsciencioAlex 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Well-documented readme ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**